packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -213,10 +200,6 @@
 			top: 1.3rem;
```

Comment:
why is `.textfield-label` changed for `.mod-select` ?

Reply:
Dead code ✔️